server/models/UserModel.py

```diff
@@ -0,0 +1,27 @@
+from app import db
+from passlib.hash import pbkdf2_sha256 as sha256
+
+class UserModel(db.Model):
+    __tablename__ = 'users'
+
+    id = db.Column(db.Integer, primary_key = True)
+    email = db.Column(db.String(120), unique = True, nullable = False)
```

Comment:
Could we have email be the primary key? It's already unique, and it's what you're using in the jwt identity.

Reply:
I have read online that it is generally a bad practice to use email as primary key for a number of reasons:
1. When we use foreign keys, we have to use email instead of id
2. Setting email as primary key means that we can never change it again.

https://stackoverflow.com/questions/3804108/use-email-address-as-primary-key

That being said, what might be the reason why you want set email as primary key?